Content.Shared/Containers/ItemSlot/ItemSlotsComponent.cs

```diff
@@ -116,6 +116,12 @@ public sealed class ItemSlot
         [ViewVariables(VVAccess.ReadWrite)]
         public bool Locked = false;
 
+        /// <summary>
+        ///     Whether the item slots system will attempt to insert this item to the user's hands when interacted with.
+        /// </summary>
+        [DataField("insertOnInteract")]
+        public bool InsertOnInteract = true;
+
```

Comment:
seems like this also blocks inserting items into the slot, not just ejecting to the hands as the comment claims

Reply:
Yeah, comment wasn't correct. Should be ok now